src/tests/ftest/ior/ior_intercept_messages.yaml

```diff
@@ -0,0 +1,53 @@
+hosts:
+    test_servers:
+        - server-A
+    test_clients:
+        - client-A
+timeout: 600
+server_config:
+    name: daos_server
+    servers:
+        log_mask: INFO
+        bdev_class: nvme
+        bdev_list: ["aaaa:aa:aa.a","bbbb:bb:bb.b"]
+pool:
+    mode: 146 # 146 is RW
+    name: daos_server
+    scm_size: 8G
+    nvme_size: 100G
+    svcn: 1
+    control_method: dmg
+container:
+    type: POSIX
+    control_method: daos
+ior:
+    client_processes:
+        np_16:
+            np: 16
+    test_file: daos:testFile
+    repetitions: 1
+    iorflags:
+        ssf:
+          flags: "-v -D 10 -w -r"
+          ior_api:
+            - POSIX
+          transfersize_blocksize:
+            1M:
```

Comment:
These can be removed and the indentation for block_size and transfer_size can be reduced by two levels as well.

Reply:
Fixed